Bigtable/src/RowMutation.php

```diff
@@ -0,0 +1,160 @@
+<?php
+/**
+ * Copyright 2018, Google LLC All rights reserved.
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *     http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+namespace Google\Cloud\Bigtable;
+
+use Google\Cloud\Bigtable\V2\MutateRowsRequest_Entry;
+use Google\Cloud\Bigtable\V2\Mutation;
+use Google\Cloud\Bigtable\V2\Mutation_DeleteFromColumn;
+use Google\Cloud\Bigtable\V2\Mutation_DeleteFromFamily;
+use Google\Cloud\Bigtable\V2\Mutation_DeleteFromRow;
+use Google\Cloud\Bigtable\V2\Mutation_SetCell;
+use Google\Cloud\Bigtable\V2\TimestampRange;
+
+/**
+ * Represents a RowMutation to perform data operation on Bigtable table.
+ * This is used to insert,update, delete operation on row in Bigtable table.
```

Comment:
Grammar: "This is used to insert, update, delete "
(Need one space after a comma)